plugins/f1-scoreboard/manager.py

```
@@ -0,0 +1,623 @@
+        upcoming["countdown_seconds"] = None
+
+        now = datetime.now(timezone.utc)
+        next_session_dt = None
+
+        for session in upcoming.get("sessions", []):
+            if session.get("status_state") == "pre" and session.get("date"):
+                try:
+                    parsed_dt = datetime.fromisoformat(
+                        session["date"].replace("Z", "+00:00"))
+                    if parsed_dt > now:
+                        next_session_dt = parsed_dt
```

Ruff (0.15.1)

[error] Local variable `next_session_dt` is assigned to but never used

Remove assignment to unused variable `next_session_dt`

(F841)